integrations/dd-sdk-android-compose/src/main/kotlin/com/datadog/android/compose/internal/Telemetry.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.compose.internal
+
+import com.datadog.android.Datadog
+import com.datadog.android.api.InternalLogger
+import com.datadog.android.api.feature.FeatureSdkCore
+
+internal fun sendTelemetry(
+    autoInstrumented: Boolean = false,
+    instrumentationType: InstrumentationType
+) {
+    val message = "$DATADOG_SEMANTICS_TELEMETRY_LOG: ${instrumentationType.name}"
+    val attributes = mapOf(
+        KEY_COMPOSE_INSTRUMENTATION to mapOf(
+            KEY_ENABLED to autoInstrumented,
+            KEY_INSTRUMENTATION_TYPE to instrumentationType
+        )
+    )
+    (Datadog.getInstance() as? FeatureSdkCore)?.internalLogger?.log(
+        level = InternalLogger.Level.INFO,
+        target = InternalLogger.Target.TELEMETRY,
+        messageBuilder = { message },
+        onlyOnce = true,
+        additionalProperties = attributes
+    )
+}
+
+internal enum class InstrumentationType {
+    Semantics,
+    ViewTracking
+}
```

Comment:
This will be obfuscated in the release build, so `${instrumentationType.name}` will give obfuscated value.

We have to define strict string literals for the values.